src/common/public/oscap_export.h

```diff
@@ -23,6 +23,8 @@
 #ifndef OSCAP_API_H
 #define OSCAP_API_H
 
+#include "oscap_platforms.h"
```

Comment:
NACK! you can't include it in a public header, this will cause fatal compiler errors to anyone using the openscap API

Reply:
@mpreisler the goal is that `oscap_platforms.h` would define the os for all the code. Is this not possible at all to do with public headers which have OS defines in them?